src/generators/static/packageFileGenerator.ts

```diff
@@ -134,8 +134,7 @@ function restLevelPackage(packageDetails: PackageDetails) {
       "unit-test": "echo skipped",
       "unit-test:node": "echo skipped",
       "unit-test:browser": "echo skipped",
-      docs:
-        "typedoc --excludePrivate --excludeExternals  --out ./dist/docs ./src"
+      docs: "echo skip"
```

Comment:
Is there a need to keep the script?

Reply:
We can remove it. Do you know if rush would complain? If not I'll get rid of it

Reply:
I don't believe rush will complain about the lack of the docs script. I don't have it in my libraries and I didn't see any complaints regarding it. Also for context, we used to have that script but we decided to remove it at some point a few months ago.

Reply:
I'll remove it